### PR TITLE
Esytech fix grammar error

### PR DESCRIPTION
**Pull request by Esyteck**
Grammatical error in the application section of the homepage was fixed. Kindly review

### CODE REVIEW

File: ols-2.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ photos:
 {% assign p-hosts = all-hosts | remove_first: ', ' | split: ", " | uniq | sort %}
 
 # The OLS-2 program
+
```
Comment (bebatut) [inline ols-2.md:54]: Could you please remove the extra line here? It will otherwise break the template

**Comment by bebatut** [comment]
Quick question: the GitHub usernames are different between the commit and the Pull Request. Is it normal?

**Comment by Esyteck** [comment]
The system I'm using now has been used by someone else so the GitHub is
interfering. I'm working on it currently though

**Comment by Esyteck** [comment]
I will remove the line now. Apologies

File: ols-2.md (hunk 1 above)
Comment (Esyteck) [reply ols-2.md]: It has been removed

**Comment by Esyteck** [comment]
I've removed the line. Kindly review ma'am

File: index.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Participants join this program with a **project** that they either are already w
 
 *Please register on Open Review before July 1, 2022 to allow activation of your Open Review profile as described in the [OLS-6 application guidelines and templates](https://github.com/open-life-science/application-forms).*-->
```
Comment (Jolah1) [inline index.md:29]: The correct word should be "Applications for OLS-7 will open at the end of 2022."
not
"Applications for OLS-7 will open ending of 2022."

**Review by Jolah1 — COMMENTED** [review]
The correct word should be "Applications for OLS-7 will open at the end of 2022."
not
"Applications for OLS-7 will open ending of 2022."
@Esyteck @bebatut

**Comment by Esyteck** [comment]
Fixed! Thanks for pointing that out

**Review by bebatut — APPROVED** [review]
Looks good now.
Thanks @Esyteck for the fix and @Jolah1 for the review

**Comment by Esyteck** [comment]
Thanks so much for the opportunity ma'am.

**Pull request merged**